app/models/job.rb

```diff
@@ -14,6 +14,14 @@ class Job < ActiveRecord::Base
   default_scope { by_newest }
 
   def self.visible_for(user)
-    all.where.not(id: user.jobs_hidden.pluck(:id))
+    _not_hidden(user)._not_liked(user)
+  end
+
```

Comment:
Could put these under `private`; even though it won't protect anything, it will signal to the next team that they aren't being called outside this file.